Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs

```diff
@@ -71,11 +71,12 @@ internal abstract class AnalyticsDataHandlerBase {
             return true;
         }
 
-        [Conditional("DEBUG_ANALYTICS")]
         private static void LogIfWarning(AnalyticsResult resp) {
+#if DEBUG_ANALYTICS
```

Comment:
I thought this had the same meaning.
What is the difference ?

Reply:
Ah, the whole execution gets culled when it's called like this
```
            LogIfWarning(
            EditorAnalytics.SendEventWithLimit(
                eventName,
                data,
                version));
```

Got it.

Reply:
Yeah exactly